extensions/widgets/browser/browser.lcb

```diff
@@ -130,7 +146,7 @@ Example:
 	liveCode.myJSHandler("myMessage", 12345);
 	
 Description:
-The <browserJavascripthandlers> is a list of LiveCode handlers that are made available to JavaScript calls within the browser. The handlers will appear as methods attached to a global "liveCode" object. You can call these methods as you would any other JavaScript function and pass whatever parameters you require.
+The <javascripthandlers> is a list of LiveCode handlers that are made available to JavaScript calls within the browser. The handlers will appear as methods attached to a global "liveCode" object. You can call these methods as you would any other JavaScript function and pass whatever parameters you require.
```

Comment:
Please include a scary-looking warning about the (pretty serious) security implications of using this with browser content loaded from sources they don't control and/or loaded over an insecure connection (i.e. not HTTPS). :scream: